src/Uno.UWP/Storage/Pickers/FileOpenPicker.iOS.cs

```diff
@@ -0,0 +1,91 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Uno.Storage.Pickers.Internal;
+using UIKit;
+using Foundation;
+using Windows.ApplicationModel.Core;
+using Uno.Helpers.Theming;
+
+namespace Windows.Storage.Pickers
+{
+	public partial class FileOpenPicker
+    {
+		private async Task<StorageFile?> PickSingleFileTaskAsync(CancellationToken token)
```

Comment:
Looks like this file uses tabs instead of spaces. Use https://marketplace.visualstudio.com/items?itemName=VisualStudioPlatformTeam.FixMixedTabs